Lib/fontParts/base/point.py

```diff
@@ -57,17 +60,17 @@ def _reprContents(self):
 
     # Contour
 
-    _contour = None
+    _contour: Optional[BaseContour] = None
 
-    contour = dynamicProperty("contour",
+    contour: dynamicProperty = dynamicProperty("contour",
                               "The point's parent :class:`BaseContour`.")
 
-    def _get_contour(self):
+    def _get_contour(self) -> Optional[BaseContour]:
         if self._contour is None:
             return None
         return self._contour()
 
-    def _set_contour(self, contour):
+    def _set_contour(self, contour: BaseContour):
```

Comment:
Setters should return `None`.

Reply:
Isn't it implied? Anyway, I'll add the return type.

Reply:
No, the implied value is `typing.Any`, and it's a good idea to specify that as well, for clarity, e.g., when annotating `**kwargs`.